vagrant-spk

```diff
@@ -411,6 +491,11 @@ STACK_PLUGINS = {
         "setup": STATIC_NGINX_SETUP_SCRIPT,
         "launcher": STATIC_NGINX_LAUNCHER_SCRIPT,
     },
+    "diy": {
+        "setup": DIY_SETUP_SCRIPT,
+        "build": DIY_BUILD_SCRIPT,
+        "launcher": DIY_LAUNCHER_SCRIPT,
```

Comment:
I'd like to see `launcher` get renamed to `start-grain` or similar, but that's not a show-stopper. I filed #15 about this wishlist desire of mine.